libgadget/hydra.c

```diff
@@ -23,26 +23,41 @@
  *  (via artificial viscosity) is computed.
  */
 
+MyFloat SPH_EOMDensity(int i)
```

Comment:
Brilliant solution! This is where I got stuck last few attempts.

Reply:
Thanks! I iterated on the name a few times